src/main/index.js

```diff
@@ -43,9 +44,11 @@ function createWindow() {
     webPreferences: {
       // Use pluginOptions.nodeIntegration, leave this alone
       // See nklayman.github.io/vue-cli-plugin-electron-builder/guide/security.html#node-integration for more info
-      nodeIntegration: process.env.ELECTRON_NODE_INTEGRATION,
+      nodeIntegration: true,
```

Comment:
this should probably be switched back to the previous `process.env.ELECTRON_NODE_INTEGRATION` unless it breaks thing in which case we should update the comment